interface-definitions/include/pki/certificate-key.xml.i

```diff
@@ -0,0 +1,23 @@
+<!-- include start from pki/certificate-key.xml.i -->
+<leafNode name="certificate">
```

Comment:
Any reason to not include `interface-definitions/include/pki/certificate.xml.i` here? Seems to be a perfekt fit.